doc/install_windows.txt

```diff
@@ -1,633 +1,177 @@
-  a ``.profile`` file in your MinGW home, containing a line like
-  ``export PATH=$PATH:/c/Python27:/c/Python27/Scripts`` (note that the latter
-  will work only when you run Theano from an MSYS shell).
+.. warning::
 
-- If you are installing the 64 bit version, you will need the following hack
-  to be able to compile Theano files with GCC (skip this step if you are using
-  the 32 bit version). In a temporary work directory, copy ``python27.dll``
-  (found in ``C:\\Windows\\System32``) as well as
-  `python27.def <http://wiki.cython.org/InstallingOnWindows?action=AttachFile&do=get&target=python27.def>`__.
-  Edit ``python27.def`` and replace ``Py_InitModule4`` with ``Py_InitModule4_64``.
-  Then open an MSYS shell, go to this temporary directory, and run:
-
-    .. code-block:: bash
-
-        dlltool --dllname python27.dll --def python27.def --output-lib libpython27.a
-
-  Finally, copy the libpython27.a file that was generated into your
-  ``C:\\Python27\\libs`` folder.
-
-- In order to run Theano's test-suite, you will need `nose
-  <http://nose.readthedocs.org/en/latest/>`__.
-  After unpacking its source code (you may use `7-zip
-  <http://www.7-zip.org/>`__), you can build and install it from within
-  its code directory by running the following command (either from a Windows
-  command prompt or an MSYS shell):
+    OpenCL support is still minimal for now.
 
-    .. code-block:: bash
-
-        python setup.py install
+Install CUDA drivers
+^^^^^^^^^^^^^^^^^^^^
 
+Follow `this link <https://developer.nvidia.com/cuda-downloads>`__
+to install the CUDA driver and the CUDA Toolkit.
 
+You must reboot the computer after the driver installation.
 
+Microsoft Visual C++ Compiler for Python 2.7 (optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+If you plan to compile libgpuarray and pygpu yourself, you will
+need to install ``Microsoft Visual C++ Compiler for Python 2.7``.  It
+provides the now-obsolete compilers form Visual Studio 2008 that are
+required for compatibility with Python 2.7. To install the package for
+all users please:
 
-Configuring the Environment
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-At this point, you should have installed all Theano dependencies.
-By default neither Python, GCC, nor Visual Studio was added to the
-PATH. Save the following shell script as ``c:\scisoft\env.bat`` to
-configure the system path:
+1. Download the installer for `Microsoft Visual C++ Compiler for Python 2.7
+   <http://www.microsoft.com/en-us/download/details.aspx?id=44266>`_.
+2. open an administrator's console (got to ``start``, then type ``cmd``,
+   right-click on the command prompt icon and select ``run as administrator``)
+3. ``cd`` to the directory when you downloaded the installer and execute ``msiexec /i
+   VCForPython27.msi ALLUSERS=1`` . The package will be installed to ``C:\Program Files
+   (x86)\Common Files\Microsoft\Visual C++ for Python\9.0``.
```

Comment:
Also note that if you are installing on Windows 7 you also want to install the Window 7 Platform SDK and activate it.  You can't activate it through the GUI because that has a bug which doesn't recognize VS 2008.  You have to do some registry hacking.

Additionally if you have a 64 bits distribution of python you need to edit the vcvarsall.bat script to properly find the 64 bit compiler because it will not work otherwise.

If you have windows 8 there is a different SDK and set of bugs that you have to contend with.  Same for Windows 10.

In light of all this complexity I would not try to give instructions on how to do that.